src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/ProcessExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+
+namespace System.Diagnostics
+{
+    public static class ProcessExtensions
```

Comment:
Extension classes need to be named after their behavior, not the type being extended. Otherwise you can have conflicts.